internal/controllers/utils/utils_test.go

```diff
@@ -859,121 +859,127 @@ nodes:
+			"installerArgs":          "[\"--append-karg\", \"nameserver=8.8.8.8\", \"-n\"]",
+			"ironicInspect":          "",
+			"role":                   "master",
+			"rootDeviceHint":         map[string]interface{}{"hctl": "1:2:0:0"},
+			"automatedCleaningMode":  "disabled",
+			"templateRefs":           []map[string]interface{}{{"name": "aci-node-crs-v1", "namespace": "siteconfig-system"}},
+			"nodeNetwork": map[string]interface{}{
+				"config": map[string]interface{}{
+					"dns-resolver": map[string]interface{}{
+						"config": map[string]interface{}{
+							"server": []string{"10.19.42.41"},
+						},
+					},
+					"interfaces": []map[string]interface{}{
+						{
+							"ipv4": map[string]interface{}{
+								"address": []map[string]interface{}{
+									{"ip": "10.16.231.3", "prefix-length": 24},
+									{"ip": "10.16.231.28", "prefix-length": 24},
+									{"ip": "10.16.231.31", "prefix-length": 24},
 								},
-								"interfaces": []map[string]interface{}{
-									{
-										"ipv4": map[string]interface{}{
-											"address": []map[string]interface{}{
-												{"ip": "10.16.231.3", "prefix-length": 24},
-												{"ip": "10.16.231.28", "prefix-length": 24},
-												{"ip": "10.16.231.31", "prefix-length": 24},
-											},
-											"dhcp":    false,
-											"enabled": true,
-										},
-										"ipv6": map[string]interface{}{
-											"address": []map[string]interface{}{
-												{"ip": "2620:52:0:10e7:e42:a1ff:fe8a:601", "prefix-length": 64},
-												{"ip": "2620:52:0:10e7:e42:a1ff:fe8a:602", "prefix-length": 64},
-												{"ip": "2620:52:0:10e7:e42:a1ff:fe8a:603", "prefix-length": 64},
-											},
-											"dhcp":    false,
-											"enabled": true,
-										},
-										"name": "eno1",
-										"type": "ethernet",
-									},
-									{
-										"ipv6": map[string]interface{}{
-											"address": []map[string]interface{}{
-												{"ip": "2620:52:0:1302::100"},
-											},
-											"enabled": true,
-											"link-aggregation": map[string]interface{}{
-												"mode": "balance-rr",
-												"options": map[string]interface{}{
-													"miimon": "140",
-												},
-												"slaves": []string{"eth0", "eth1"},
-											},
-											"prefix-length": 64,
-										},
-										"name":  "bond99",
-										"state": "up",
-										"type":  "bond",
-									},
+								"dhcp":    false,
+								"enabled": true,
+							},
+							"ipv6": map[string]interface{}{
+								"address": []map[string]interface{}{
+									{"ip": "2620:52:0:10e7:e42:a1ff:fe8a:601", "prefix-length": 64},
+									{"ip": "2620:52:0:10e7:e42:a1ff:fe8a:602", "prefix-length": 64},
+									{"ip": "2620:52:0:10e7:e42:a1ff:fe8a:603", "prefix-length": 64},
```

Comment:
We shouldn't be using real IP addresses in unit tests. Would prefer to used the addresses reserved for documentation see https://en.wikipedia.org/wiki/Reserved_IP_addresses